### ISSUE TITLE
[Bug] font weight doesnt apply, no custom fonts

### ISSUE THREAD

**Issue by jonasgroendahl** [jsartisan/react-native-magnus#67]
## Description

I tried to add font-weight to a text element and it didn't work. I'm not using any custom fonts.

### Steps to reproduce the behaviour:

import { Text } from 'react-native-magnus';

```
  <Text fontWeight="bold" fontSize="4xl" mt="md">
          Iphone 11
        </Text>
```

### Important Details

- Version:   "react-native-magnus": "^1.0.59",   "react-native": "0.63.4",
- OS: Mac
- Browser: iOS simulator

**Comment by jsartisan** [comment]
@jonasgroendahl Fixed in v1.0.60

**Comment by ayrtonlacerda** [comment]
Don't Work in iOS 14 - iphone 11 Pro simulator

version 1.0.62

![Screen Shot 2021-12-26 at 20 06 42](https://user-images.githubusercontent.com/38504590/147422092-c48f57cb-f07f-4b4d-b6b1-89b56268d0e7.png)
2
